### PR TITLE
Added Author type Facet to Browse Filters EXLM-1352

### PR DESCRIPTION
**Pull request by nitin-rachabathuni**
Please always provide the Jira Issue your PR is for, as well as test URLs where your change can be observed (before and after):

Jira ID: https://jira.corp.adobe.com/browse/EXLM-1352

Test URLs:

- Before: https://main--exlm--adobe-experience-league.hlx.live/en/browse
- After: https://main--exlm--adobe-experience-league.hlx.live/en/browse?martech=off

### CODE REVIEW

**Review by noormd08 — CHANGES_REQUESTED** [review]
Lets connect and discuss on the changes.

File: blocks/browse-filters/browse-filter-utils.js [hunk 1]
```diff
@@ -66,6 +66,12 @@ const contentType = [
     title: 'Certification',
     description: 'Credentials that recognize an individual’s skill and competency in an Adobe solution.',
   },
+  {
+    id: 'Article',
+    value: 'Article',
+    title: 'Article',
+    description: 'Article that recognize an individual’s skill and competency in an Adobe solution.',
+  },
```
Comment (noormd08) [inline blocks/browse-filters/browse-filter-utils.js:74]: These text should be loaded with placeholders.

File: blocks/browse-filters/browse-filter-utils.js [hunk 2]
```diff
@@ -145,6 +151,23 @@ const expLevel = [
...
+  {
+    id: 'external',
+    value: 'External',
+    title: 'External',
+    description:
+      'High degree of proficiency with an advanced understanding of concepts and skill. Regularly manages complex tasks and objectives.',
+  },
+];
+
```
Comment (noormd08) [inline blocks/browse-filters/browse-filter-utils.js:170]: Same as above.

File: scripts/data-service/coveo/coveo-token-service.js [hunk 3]
```diff
@@ -121,6 +121,12 @@ export default async function loadCoveoToken() {
     coveoResponseToken ||
     // eslint-disable-next-line no-async-promise-executor
     new Promise(async (resolve) => {
+      if (window.location.window !== 'experienceleague.adobe.com') {
+        // Token allows acces to staging search functionality, but not analytics
+        const { coveoToken } = getConfig();
+        resolve(coveoToken);
+        return;
+      }
```
Comment (noormd08) [inline scripts/data-service/coveo/coveo-token-service.js:129]: Is this specific to Article Cards? If so, we may need to tweak a bit.

File: blocks/browse-filters/browse-filter-utils.js (hunk 2 above)
Comment (nitin-rachabathuni) [reply blocks/browse-filters/browse-filter-utils.js:176]: done done Noor

File: scripts/coveo-headless/engine.js [hunk 4]
```diff
@@ -1,13 +1,15 @@
+import loadCoveoToken from '../data-service/coveo/coveo-token-service.js';
 import { getConfig } from '../scripts.js';
 import { generateCustomContext, generateMlParameters, COVEO_SEARCH_CUSTOM_EVENTS } from '../search/search-utils.js';
 
-export default function buildHeadlessSearchEngine(module, coveoToken) {
+export default async function buildHeadlessSearchEngine(module) {
   const { coveoOrganizationId } = getConfig();
+  const coveoTokenString = await loadCoveoToken();
```
[the remark this answers is not in the document]
Comment (nitin-rachabathuni) [reply scripts/coveo-headless/engine.js]: done done Noor

File: scripts/data-service/coveo/coveo-token-service.js (hunk 3 above)
Comment (nitin-rachabathuni) [reply scripts/data-service/coveo/coveo-token-service.js:130]: These are temporary changes, so added a comment Noor.

**Review by noormd08 — APPROVED** [review]

**Pull request merged**